src/components/Listing/Listing.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import './Listing.css';
+
+const Listing = (props) => {
+
+  const renderListingDetails = () => {
```

Comment:
Breaking change: tried to use Object.keys to iterate over specifically a given listing's "details" object (where beds, baths, cost per night are). But having some weird behavior, its showing the information for EVERY listing, underneath every single listing (if that makes sense - more detailed descrip / example in the PR).